src/main/java/biz/paluch/spinach/api/rx/DisqueJobReactiveCommands.java

```diff
@@ -45,45 +47,60 @@
     Observable<String> addjob(K queue, V job, long timeout, TimeUnit timeUnit, AddJobArgs addJobArgs);
 
     /**
-     * Get jobs from the specified queue. By default COUNT is 1, so just one job will be returned. If there are no jobs in any
-     * of the specified queues the command will block.
+     * Get jobs from the specified queue. If there are no jobs in any of the specified queues the command block until
+     * given timeout and return null.
      *
-     * @param queue the queue
-     * @return the job
+     * @param queues queue names
+     * @return the job or null
      */
-    Observable<Job<K, V>> getjob(K queue);
+    Observable<Job<K, V>> getjob(K... queues);
```

Comment:
:+1: Improves the API. Could you add `@Deprecated` to the corresponding `getjobs(K... queues)` method?